books/kestrel/apt/isodata-doc.lisp

```diff
@@ -397,6 +397,15 @@
       (xdoc::li
        "@(':auto'), to use @('nil') or @('(mv nil ... nil)') for single-value
         and multi-value functions respectively.")
+      (xdoc::li
+       "@(':base-case'), to search for a base-case within the domain of the new
```

Comment:
I would make the two options more symmetric, i.e. `:base-case-then` and `:base-case-else`, also switching them in the two `xdoc::li`s (i.e. first describing `:base-case-then`, then `:base-case-else` briefly).